shared/uavobjectdefinition/stateestimation.xml

```diff
@@ -11,6 +11,7 @@
 			</description>
       <options>
         <option>Complementary</option>
+        <option>Complementary+VelCompass</option>
```

Comment:
Please add brief description to field description.

Reply:
Done, thx.

Reply:
Great 👍